tests/Agent/IntegrationTests/Applications/AzureFunctionApplication/HttpTriggerFunctionUsingAspNetCorePipeline.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2020 New Relic, Inc. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+#if NET9_0
```

Comment:
Maybe a comment or something to indicate why this is necessary? When we do our annual update for new .NET versions, it isn't always clear which `#ifdef`s need to be updated